raiden/network/rpc/client.py

```diff
@@ -129,10 +129,19 @@ def geth_discover_next_available_nonce(web3: Web3, address: AddressHex) -> Nonce
 
 
 def check_address_has_code(
-    client: "JSONRPCClient", address: Address, contract_name: str = "", expected_code: bytes = None
+    client: "JSONRPCClient",
+    address: Address,
+    contract_name: str = "",
```

Comment:
Also here: all usages I find set `contract_name`, so I'd also remove the default here. But we can also do this in a follow-up.